web/libs/editor/src/components/InteractiveOverlays/CommentsOverlay.tsx

```diff
@@ -152,10 +210,15 @@ const CommentsOverlayInner: React.FC<CommentsOverlayProps> = observer(({ annotat
   return (
     // biome-ignore lint/a11y/noSvgWithoutTitle: It's not just an icon or a figure; it's an entire interactive layer.
     <svg className={containerStyles.join(" ")} ref={setRef} xmlns="http://www.w3.org/2000/svg">
-      {overlayComments.map((comment: MSTComment) => {
-        const { id } = comment;
-        return <CommentItem key={[id, uniqKey].join("-")} comment={comment} rootRef={rootRef} />;
-      })}
+      <g key={uniqKey}>
+        {annotation.isLinkingMode && annotation.results.map((result: MSTResult) => (
+          <ResultTagBbox key={result.id} result={result} rootRef={rootRef} />
+        ))}
+        {overlayComments.map((comment: MSTComment) => {
+          const { id } = comment;
```

Comment:
previously there was the need for `uniqKey` we dont have that anymore - is that fixed some where? or just missed?

Reply:
It was done to be sure it would be re-rendered. It was moved slightly above.
(line 213)
```html
<g key={uniqKey}>
```

https://github.com/HumanSignal/label-studio/pull/6486/files/d25e0231036c38a02426180aa3f44723fe6dfc04#diff-b0872dfdbd0fb99147d386c065deadb2df6c78d7ed3da7b440b9aa0b01ee0e36R213